### ISSUE TITLE
Inquiry about the pre-trained Mask R-CNN checkpoints

### ISSUE THREAD

**Issue by yeda318** [alfworld/alfworld#28]
Hi,
Thanks for your interesting work and the release of the source codes!

I'm trying to use the pre-trained Mask R-CNN checkpoints you've published via [Google Drive](https://drive.google.com/drive/folders/1eHzFw-a7IIX5iV-tQTu6fXKIuxxha6ag?usp=sharing),
but based on what I've examined, it seems that the checkpoint `mrcnn_alfred_receptacles_004.pth` is associated with 73 classes like `mrcnn_alfred_objects_sep13_004.pth`, whereas `constants.STATIC_RECEPTACLES` contains 32 classes:

![image](https://user-images.githubusercontent.com/48595269/234576987-ca8f9568-667b-4adc-b613-71bfccaa5859.png)

I'm getting an error if I call `load_state_dict()` of the Mask R-CNN model created with `len(constants.STATIC_RECEPTACLES) + 1` as an argument.
Would there be any possibility that the checkpoint `mrcnn_alfred_receptacles_004.pth` is in fact for `constants.OBJECTS_DETECTOR` rather than `constants.STATIC_RECEPTACLES`?

I would really appreciate it if you could help me with this issue.
Thank you!

**Comment by MohitShridhar** [comment]
@yeda318, good catch. It's been a while, maybe I uploaded the wrong file. 

You probably have to train the receptacle detector yourself with [this script](https://github.com/alfworld/alfworld/blob/master/scripts/train_mrcnn.py).

**Comment by MohitShridhar** [comment]
@yeda318, I found the right file: https://github.com/alfworld/alfworld/releases/download/0.2.2/mrcnn_alfred_receptacles_sep13_007.pth

``` python
d = torch.load('mrcnn_alfred_receptacles_sep13_007.pth')
d['roi_heads.box_predictor.cls_score.weight'].shape
torch.Size([33, 1024])
```

I don't know if the checkpoint is good, but I hope this helps!

**Comment by runamu** [comment]
@MohitShridhar, thank you for your prompt response and for providing the necessary file. It appears to be what I was looking for, and I believe it will greatly benefit our project. Thanks again!
